packages/fiori-freestyle-writer/test/__snapshots__/basic.test.ts.snap

```diff
@@ -33,12 +33,12 @@ archive.zip
     \\"@sap/ux-ui5-tooling\\": \\"1\\"
   },
   \\"scripts\\": {
-    \\"start\\": \\"fiori run --open 'test/flpSandbox.html#nods1-tile'\\",
-    \\"start-local\\": \\"fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html#nods1-tile'\\",
+    \\"start\\": \\"fiori run --open 'test/flpSandbox.html?data-sap-ui-xx-viewCache=false#nods1-tile'\\",
```

Comment:
@slin-sap I believe the query parameter is `sap-ui-xx-viewCache`. 
`data-sap-ui-xx-viewCache` is for use in the script tag of the html file.

Reply:
Thanks